CONTRIBUTING.md

```diff
@@ -25,6 +25,9 @@ gulp
 
 When you're ready, continue with next steps:
 
+- Run ```sh
+gulp uglify
```

Comment:
This is for the minification. Not for the tests.

Em seg, 20 de jul de 2015 às 19:16, Fernando Daciuk <
notifications@github.com> escreveu:

> In CONTRIBUTING.md
> <https://github.com/fdaciuk/ajax/pull/1#discussion_r35050840>:
>
> > @@ -25,6 +25,9 @@ gulp
> >
> >  When you're ready, continue with next steps:
> >
> > +- Run ```sh
> > +gulp uglify
>
> Actually, you need to run just gulp, or gulp webserver test lint to run
> tests and assure that the code has no lint errors.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/fdaciuk/ajax/pull/1/files#r35050840>.
>
-- 
Sent from my mobile